docs/only-allow-admins-to-create-new-streams-feature.md

```diff
@@ -0,0 +1,12 @@
+# Only allow admins to create new streams
+
+1. Login with an Admin account.
+2. Locate the cog
+(![cog](/static/images/help/cog.png)) on the main navigation bar and click on
+it to relase the dropdown menu.
+![Settings dropdown](/static/images/help/settings-dropdown-menu-administration.png)
+3. Select the Administration option. This will open the Administration page
+of your organization.
+4. Under Organization settings select the **Only admins may create streams** checkbox.
+![Organization settings](/static/images/help/administration-organization-settings.png)
```

Comment:
OK, this screenshot is really large and shows a lot of irrelevant information. Can you try and crop it to show only the **Organization settings** box?

If you want a nice screenshot taker try using Lightshot Screenshot.